Lib/shlex.py

```diff
@@ -313,7 +313,14 @@ def split(s, comments=False, posix=True):
 
 
 def join(split_command):
-    """Return a shell-escaped string from *split_command*."""
+    """Return a shell-escaped string from *split_command*.
+
+    This function is the inverse of :func:`split`.
+
+    >>> import shlex
+    >>> print(shlex.join(['echo', '-n', 'Multiple words']))
+    echo -n 'Multiple words'
```

Comment:
Thanks! As a user what I was really looking for was something saying explicitly that `shlex.join` takes an `argv` and turns it into something I can paste into a shell command. I know I suggested `echo` as an example, but it seems like `printf '%s\n'` is better. Maybe this?:
```suggestion
    >>> import shlex
    >>> print(shlex.join(['printf', '%s\n', 'foo', 'hello world', 'bar']))
    printf '%s
    ' foo 'hello world' bar
    >>>
    $ printf '%s
    ' foo 'hello world' bar
    foo
    hello world
    bar
    $
```
I find it surprisingly hard to articulate what this does in words.